code/modules/mob/holder.dm

```diff
@@ -44,12 +44,12 @@
 	src.forceMove(H)
 	H.name = name
 	if(desc)	H.desc = desc
-	H.attack_hand(grabber)
+	H.MouseDrop(grabber)
```

Comment:
Er, this.. shouldn't actually be.. that. It's definitely intended to be attack_hand, because it's calling holder.attackhand. I don't think this actually does anything for that matter, but it still shouldn't be changed.